internal/backend/deploy.go

```diff
@@ -573,13 +574,24 @@ func (in *instance) addPreArchives(tainr *types.Container, pod *corev1.Pod) erro
 				LocalObjectReference: corev1.LocalObjectReference{
 					Name: cm.ObjectMeta.Name,
 				},
+				Items: func() []corev1.KeyToPath {
+					items := []corev1.KeyToPath{}
+					for file, FileInfo := range pfiles {
```

Comment:
Can you rename FileInfo to info for conciseness, within the current context the name makes its intent clear enough.